website/docs/r/group_profile.html.markdown

```diff
@@ -0,0 +1,47 @@
+---
+layout: "turbot"
```

Comment:
What is the layout field?

Reply:
layout  - Which navigation sidebar to display for this page. A layout called <NAME> gets loaded from ./content/source/layouts/<NAME>.erb.

In our case NAME is turbot